tests/test_token.py

```diff
@@ -1,33 +1,29 @@
 """Test for word tokenizers"""
-import unittest
-
 from tiny_tokenizer.tiny_tokenizer_token import Token
 
 
-class TokenTest(unittest.TestCase):
-    """Test ordinal word tokenizer."""
+def test_token_without_feature():
+    token = Token(surface="大崎")
+    assert"大崎" == token.surface
```

Comment:
There isn't space between `assert` and string.

Reply:
Fixed in 62d68d8. :pray: